src/main/java/net/fabricmc/loom/mixin/ObfuscationServiceFabric.java

```diff
@@ -69,6 +70,11 @@ private void addSupportedOptions(ImmutableSet.Builder<String> builder, String fr
 	}
 
 	@Override
```

Comment:
Not sure if this needs to be gone? for 0.6 support? Does java check @Override at runtime? im not sure.

Reply:
![image](https://user-images.githubusercontent.com/7806504/68534582-ca29b800-02fb-11ea-90b3-bccb8504b212.png)